### DISCUSSION TITLE
Issue: Unexpected error while connecting to the machine

### DISCUSSION THREAD

**Discussion by DarkSide1305** [cockpit-project/cockpit#17262] [Q&A] [answered]
Edit: Solved! Can mask as answered or delete.

Hello. So im running Cockpit on Ubuntu 20.04.4 LTS and using a reverse proxy via Nginx. Connecting to it via my domain works with it but whenever I try to log in it tells me there is an unexpected error, telling me to check my journalctl, but theres no issue. Nginx has nothing and journalctl just tells me the that it started the server

**Comment by martinpitt** [comment, marked as answer]
Thanks! Closing then.

**Comment by rajivvishwa** [comment]
> Edit: Solved! Can mask as answered or delete.

How did you solve this? I've the same issue.

**Reply by fschiefelbein** [reply]
I had the same issue. I've solved it allowing websocket connection in the Nginx reverse proxy.

**Reply by fschiefelbein** [reply]
proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection "upgrade";
